iceoryx_examples/singleprocess/single_process.cpp

```diff
@@ -121,9 +121,11 @@ int main()
     //! [roudi config]
 
     //! [roudi]
-    iox::roudi::RouDi roudi(roudiComponents.rouDiMemoryManager,
-                            roudiComponents.portManager,
-                            iox::roudi::RouDi::RoudiStartupParameters{iox::roudi::MonitoringMode::OFF, false});
+    constexpr bool DO_NOT_TERMINATE_APP_IN_ROUDI_DTOR = false;
+    iox::roudi::RouDi roudi(
+        roudiComponents.rouDiMemoryManager,
+        roudiComponents.portManager,
+        iox::roudi::RouDi::RoudiStartupParameters{iox::roudi::MonitoringMode::OFF, DO_NOT_TERMINATE_APP_IN_ROUDI_DTOR});
```

Comment:
The parameter in `RoudiStartupParameters` is named `killProcessesInDestructor`. I didn't check what it actually does but the name suggests that we should name the constant `DO_TERMINATE_APP_IN_ROUDI_DTOR`.

Reply:
Maybe, the idea was to state the intention in the name instead of the content.

Reply:
Since it is set to false, it reads for me like we want to terminate all apps when RouDi goes out of scope... But maybe it's just me. When the other reviewer is okay with this, I'll be too.

Reply:
I also always have this misunderstanding. Lately I added `indicator` to the constant, e.g. `DO_TERMINATE_APP_IN_ROUDI_DTOR_INDICATOR = false`. In general I prefer enums though.

Reply:
@FerdinandSpitzschnueffler @elBoberido So tell me, what you would like to read here and I am happy to change it.

Reply:
What about `TERMINATE_APP_IN_ROUDI_DTOR_FLAG`, set to false?